.github/workflows/release.yml

```diff
@@ -26,5 +26,7 @@ jobs:
 
       - name: Run chart-releaser
         uses: helm/chart-releaser-action@v1.1.0
+        with:
+          charts_dir: charts
```

Comment:
Unfortunately we have to use the "chart" directory, instead of the "charts" directory because some people are using the docker-hub-rate-limit-exporter by cloning this repository in their CI-Pipeline. If move all files to "charts", we would break their CI-Pipelines.

Reply:
Resolved with https://github.com/viadee/docker-hub-rate-limit-exporter/pull/22/commits/c76f7f8c4c218664b2e1c94ced00cddcf5e3352a